### PR TITLE
[mui-image] add types

### PR DESCRIPTION
**Pull request by benmneb**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## mui-image ([<kbd>unpkg</kbd>](https://unpkg.com/browse/mui-image@latest/))
was missing the following properties:
1. Image
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f55dbfc4066b96ee70b9b10c79f5042a5603d003
</p>

File: types/mui-image/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    onLoad?: (...args: any[]) => any;
+    position?: React.CSSProperties['position'];
+    shift?: 'left' | 'right' | 'top' | 'bottom' | false | null;
+    shiftDuration?: number;
+    showLoading?: boolean | React.ReactNode;
+    src: string;
+    style?: React.CSSProperties;
+    width?: React.CSSProperties['width'] | number;
+    wrapperClassName?: string;
+    wrapperStyle?: React.CSSProperties;
```
Comment (peterblazejewicz) [inline types/mui-image/index.d.ts:31]:
```suggestion
    alt?: string | undefined;
    bgColor?: React.CSSProperties['backgroundColor'] | undefined;
    className?: string | undefined;
    distance?: string | number | undefined;
    duration?: number | undefined;
    easing?: React.CSSProperties['transitionTimingFunction'] | undefined;
    errorIcon?: boolean | React.ReactNode | undefined;
    fit?: React.CSSProperties['objectFit'] | undefined;
    height?: React.CSSProperties['height'] | number | undefined;
    iconWrapperClassName?: string | undefined;
    iconWrapperStyle?: React.CSSProperties | undefined;
    onError?: (...args: any[]) => any | undefined;
    onLoad?: (...args: any[]) => any | undefined;
    position?: React.CSSProperties['position'] | undefined;
    shift?: 'left' | 'right' | 'top' | 'bottom' | false | null | undefined;
    shiftDuration?: number | undefined;
    showLoading?: boolean | React.ReactNode | undefined;
    src: string;
    style?: React.CSSProperties | undefined;
    width?: React.CSSProperties['width'] | number | undefined;
    wrapperClassName?: string | undefined;
    wrapperStyle?: React.CSSProperties | undefined;
```
would be nice to have support for TS4.4 exact optional property types here:
https://www.typescriptlang.org/play?exactOptionalPropertyTypes=true&ts=4.4.2&ssl=1&ssc=1&pln=2&pc=1#code/PTAEHUEsBcAtQKYA8CGBjaB5ADtSB7AOxQBsAFAJ32wQugE8AVemgZ0WICMSEATAGlDMaAZTQVIuALAAoEKFgp2raBIy1QFAK492KKlsK9QAd0XRQ9fFtBoUhUKwQX7oaniKlZ87FRp16UGh8ADpZbzAAMXwKRFQAW2weQThIdkhCaFoAM3QEUF4ENBJ9BHY4FAs4Wny00BQI0F9qWgZTWEg0eDsHTnyifuygk3xHVQyAc1YALlAAcl59AGs5t1i5kkgJ2Gg5xpjQGEdYaxJjQnwLPsOHardOACsi6DCZWQysily0fIBVJwoABEELkdNB2ABvWSgUDyRiwfIoThOQg-NxDFCgABupC0+QoCF8ZQQmXYc1Y9BUCHiexkMLQ+BIMXh1IQmCxtAkhQA-LMAESLChLPmgAA+oD5m220D5AG5ZABfcJyMBQODWKoddjZEoTDhIngCIIIgn1U1wAn5HEkPHsMydeBWGw9RpOCwAAwZTIoLPibI5FC5CHdQVGfX5guFgklWx2IvsxndhkK2QyfHdr0aImceEIeru1rxodASaMILTvBDJkgJBI9Vr+BMoHi+BUoAAUigcWIJLhNIY8H7Gl0ikttQc7sg0llUflgqBcjXBPauod2CDss9IBySIFciQKa95AAJRsIAMpE0Icn1BdaABe97TrHKo1S7D6iixBAoISECIKctCBgAhCEabIDk9RlmQRP12U5SBChDAB9Xh8DKQg5mgZCbmKLRChLZMgPTTMVX-fIdRQPV3WQdAsFwUDSEoFoAmEMoQ3iFAljKIQWAQHtJBcNA0C0ChKgQXdGk46BVzuFMMhAogmiob9CmMJR6gcdxGLrZp-AYUjCmKUo71RDwHAmZx-lobNoFzKYAAoAEpZmsoEQRQMFWHlN4ZAZQg2zdez2AAXlASzoDc2zgucnygsmVgQi9GDWXgwNEPyMKBWWOVZHivNEuSn1YP9BCCKyqU4x8xoACEtAsC5oGmPKcwSpLoOK1KAyDUAwqI1NCD4aqyMoH9izuSjqNojAcHMpi-FaJg+NYd1LzqEwNMappW1YSBuAQEIgA

File: types/mui-image/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+declare const Image: React.FC<MuiImageProps>;
+export default Image;
```
Comment (peterblazejewicz) [inline types/mui-image/index.d.ts:35]:
```suggestion
export default Image;
export { Image };
```

File: types/mui-image/mui-image-tests.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import Image from 'mui-image';
+
+(function Test() {
+    function onLoad() {
+        console.log('loaded!');
+        return 42;
+    }
+
```
Comment (peterblazejewicz) [inline types/mui-image/mui-image-tests.tsx:9]:
```suggestion
```
stay focused :)

File: types/mui-image/index.d.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+// Type definitions for mui-image 1.0
+// Project: https://github.com/benmneb/mui-image
+// Definitions by: benmneb <https://github.com/benmneb>
+//                 Natalia <https://github.com/CodeMeNatalie>
```
Comment (peterblazejewicz) [inline types/mui-image/index.d.ts:4]: You have to this person to ACK being added here as owner, as the automation sends notification to onwers handles (so effectively emails owners). Sorry for the inconvience!

Comment (peterblazejewicz) [reply types/mui-image/index.d.ts:4]: sorry, please do not skip this one

Comment (peterblazejewicz) [reply types/mui-image/index.d.ts:4]: You can give this user credits due as inline comment, but do not put into owners, as autmation will send notifications

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@benmneb thanks!

**Comment by benmneb** [comment]
Ready to merge

**Pull request merged**